<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSpinBox" name="NoteLeistungskurs">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NoteDeutsch">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NoteEnglisch">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NoteMathe">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="Deutsch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mathe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mathe</string>
   </property>
  </widget>
  <widget class="QLabel" name="Englisch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Englisch</string>
   </property>
  </widget>
  <widget class="QLabel" name="Leistungskurs">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Leistungskurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="berechnen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ergebnis">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
